pandas/core/arrays/interval.py

```diff
@@ -790,6 +790,36 @@ def argsort(
             ascending=ascending, kind=kind, na_position=na_position, **kwargs
         )
 
+    def min(self, *, axis: int | None = None, skipna: bool = True):
+        if axis is not None and axis >= self.ndim:
+            raise ValueError(axis)
```

Comment:
case hit in tests?

Reply:
Updated with test, also fixes the mypy complaint on master